### PR TITLE
Add hello world application for flex deploy check

### PR DESCRIPTION
**Pull request by nkubala**
This is taken from https://github.com/GoogleCloudPlatform/nodejs-docs-samples, and is used to ensure basic flex deploys work as intended.

@ofrobots

### CODE REVIEW

File: base/test/deploy_check/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "appengine-hello-world",
+  "description": "Simple Hello World Node.js sample for Google App Engine Flexible Environment.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
```
Comment (DominicKramer) [inline base/test/deploy_check/package.json]: Do we want this URL to point to the `nodejs-docker` repo here?

**Comment by nkubala** [comment]
I don't think we want this to run whenever an image is built; this would increase build times by at least 5 minutes, and IMO isn't really necessary. This job will run on a cron, probably nightly.

File: base/test/deploy_check/package.json (hunk 1 above)
Comment (nkubala) [inline base/test/deploy_check/package.json]: possibly? I copied this code directly from that repo, but can change it if you want.

**Comment by DominicKramer** [comment]
As long as a test failure is due to a failure of the image, and not the test itself, the benefits of the tests, I think, are worth waiting a few minutes.  If possible, it would be nice to have a flag to disable the tests to ease local testing.

I am concerned that a built image will pass all other tests, but not until hours later will the cron job find a problem.  I would rather wait longer upfront when the image is being built.

**Comment by nkubala** [comment]
Keep in mind that this test also tests the builder flow, not just the integrity of the base image. I agree that there should be rigorous testing on the base image itself, but this test should be decoupled from the image build. Any built image will always be put through the gauntlet before being released.

**Comment by ofrobots** [comment]
Starting with cron sounds like a good starting point for now. Once that is running stably, it would be good to promote this to a test that gets run on a PR merge to catch problems sooner.

**Pull request merged**